mslib/msui/constants.py

```diff
@@ -33,6 +33,7 @@
 
 # ToDo refactor to generic functions, keep only constants
 HOME = os.path.expanduser(f"~{os.path.sep}")
+DOWNLOAD_PATH = os.getenv("MSUI_CONFIG_PATH", os.path.join(HOME, ".config", "msui", "downloads", "aip"))
```

Comment:
can you move this into the other module, because it is specific for aip in airdata only. It won't be used anywhere else.